Document minimal billing unit (#1030)
flowforge/flowforge: docs/cloud/billing.md

## Removing Projects

> If you delete a project but still have other projects on their account you will receive a pro-rated credit for the number of seconds remaining in the billing cycle, which is used against a future invoice.

Reviewer: ```suggestion
When a project is deleted you will receive a pro-rated credit for the number of seconds remaining in the billing cycle, which is used against a future invoice.
```

Reviewer: I got confused by the if statement, what happens if the team is empty?

Author: there's currently no difference if its the last project on the team or there are others, the account gets a credit and it sits on the account until there's an invoice.
We can manually  refund credit if they close their account and request it

## Suspended Projects

> You are not billed for suspended projects.

Reviewer: ```suggestion
Suspended projects have no running editor, nor a runtime. You are not charged for suspended projects.
```

> When you suspend a project your account will receive a credit for the number of seconds remaining in the billing cycle.

Reviewer: ```suggestion
When a project is suspended your account will receive a credit for the number of seconds remaining in the billing cycle.
```
